index.html

```diff
@@ -752,6 +790,17 @@ <h3>
         <h3>
           Presentation initiation by the controlling UA example
```

Comment:
Starting a presentation from the controlling user agent

Reply:
Done